### PR TITLE
feat/Dp-4959 added handle different network circuits feature flag

### PR DESCRIPTION
**Pull request by Abbath1349**

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -395,16 +395,21 @@
...
       "default": false
     },
-    "targeting.schema" : {
+    "targeting.schema": {
       "type": "string",
       "title": "Schema of targeting form",
       "description": "JSON with description of UI schema for targeting form",
       "default": null
+    },
+    "file.is_load_from_circuits_enabled": {
```
Comment (nkt) [inline docs/feature-flags.schema.json]: should be in camel-case

Comment (nkt) [reply docs/feature-flags.schema.json]: And I think this flag is not needed.

Comment (Abbath1349) [reply docs/feature-flags.schema.json]: Fixed. Changed to camelCase

Comment (Abbath1349) [reply docs/feature-flags.schema.json]: Oleg Shurupin said: this flag is needed to understand show this error to user or no.

**Pull request closed without merging**